docs/getting-started/examples.md

```diff
@@ -240,6 +240,29 @@ with Diagram("Broker Consumers", show=False):
     queue = Custom("Message queue", rabbitmq_icon)
 
     queue >> consumers >> Aurora("Database")
-````
+```
+
+![rabbitmq consumers diagram](/img/rabbitmq_consumers_diagram.png)
+
+## Diagram with Custom Edges
```

Comment:
This example is not practical or meaningful, and too long to show in the example page. Please give a more realistic example like https://diagrams.mingrammer.com/docs/getting-started/examples#advanced-web-service-with-on-premise

Or, you can just modify the exiting examples with customized edges.

Reply:
Done